docs/data-apps/ecommerce-analytics/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "E-commerce Analytics"
+sidebar_position: 2
+sidebar_label: "E-commerce Analytics"
+---
+
+:::caution
+
+This data app is currently in Public Preview and features may change without notice. 
+
+:::
+
+## Introduction
+
+<!-- Ecommerce reports based on GA4's Monetization metrics, covering revenue, purchases, checkout journey and promotions -->
+
+This application offers an in-depth breakdown of Ecommerce activities across your websites and mobile apps, presenting an enhanced rendition of Google Analytics v4 reports.
+Access real-time data directly within your data warehouse in a governed manner.
+Navigate through a series of interactive dashboards designed to surface monetization data, purchasing behaviors, checkout experiences, and transaction patterns.
+Customize your analytics workflow with filters for campaigns, channels, and products.
+
+Leveraging Snowplow's latest Ecommerce event tracking and our proprietary E-commerce dbt package, the app transforms raw event data into structured tables covering carts, checkouts, products, transactions, and user interactions, providing valuable insights for informed decision-making.
+
+![](images/overview.png)
+
+### Requirements
```

Comment:
Perhaps add that they also need to track using our latest Ecommerce plugin, and link the web + mobile plugins?